apps/codeforafrica/.env

```diff
@@ -0,0 +1,2 @@
+NEXT_PUBLIC_IMAGE_DOMAINS="res.cloudinary.com"
+NEXT_PUBLIC_APP_URL="http://localhost:3002"
```

Comment:
Not sure why git picked your .env file?

Reply:
@gertie-sheshe Forgot to remove it...i was using it for testing purposes